### ISSUE TITLE
Labels/Hashtags for Transactions

### ISSUE THREAD

**Issue by d3xter** [AndreAle94/moneywallet#96]
I'd like to add multiple (0...*) Labels/Hashtags/... to Transactions.

For example I'd like to track whether Lunch was Business related without having to duplicate the Categories

**Comment by d3xter** [comment]
If you think it's a viable feature, I'd be willing to implement it

**Comment by AndreAle94** [comment]
Yes, this could be a nice feature to have. How do you plan to implement it? However, you can fork the project and create a pull request. At the moment i have no time to add new features.

**Comment by d3xter** [comment]
My Plan is to implement it similar to People/TransactionPeople without modifying existing code as much as possible
